tests/cypress/e2e/for-teacher_page/class_page/create_accounts_page/generate_password.cy.js

```diff
@@ -0,0 +1,20 @@
+import {loginForTeacher} from '../../../tools/login/login.js'
+import {createClass} from '../../../tools/classes/class.js'
+
+
+describe('Is able to go to logs page', () => {
+  it('Passes', () => {
+    loginForTeacher();
+    cy.wait(500);
+    createClass();
+    cy.get(".view_class").first().click(); // Press view class button
+    cy.get('#add-student').click();
+    cy.get('#create-accounts').click(); 
+    cy.get('#toggle_circle').click();
```

Comment:
Toggle circle is a bit vague. If the id was already there, our bad! But could you maybe explain what it does or call the element another way? Another contributor is unable to understand this line.